transport/src/main/java/io/netty/channel/ChannelOutboundBuffer.java

```diff
@@ -829,4 +810,25 @@ Entry recycleAndGetNext() {
             return next;
         }
     }
+
+    private static final class ChannelWritabilityChangedTask implements Runnable {
+        private final Channel channel;
+        private boolean writable = true;
```

Comment:
Really? When a channel is inactive (both that went inactive from active and that never went active), isWritable() returns false.